docs/_data/portland-2021-schedule.yaml

```diff
@@ -0,0 +1,73 @@
+writing_day:
+  - time: '8:00'
+    title: 'Platform opens'
+  - time: '9:30'
+    title: Morning introduction - Introduction to Sprints & Project Introductions
+  - time: '9:45'
+    title: Team up and start working
+  - time: '12:30'
+    title: Lunch Break
+  - time: '5:00'
+    title: End of Writing Day
+
+talks_day1:
+  - time: '8:00'
+    title: 'Platform opens'
+  - time: '9:00'
+    title: Write the Docs Team - Introduction to Write the Docs
+  - time: '9:45'
+    title: "<b>Unconference Starts</b>"
+  - time: '9:45'
+    slug: writing-a-perfect-technical-tutorial-jessica-garson
+  - time: '10:30'
+    slug: shuffle-ball-change-sashay-your-way-to-clearer-api-documentation-rachael-stavchansky
```

Comment:
I think we should have a break, even for the virtual conference. Otherwise it feels a bit overwhelming.

Reply:
Done